### PR TITLE
Add 8.13 what’s new

### PR DESCRIPTION
**Pull request by bmorelli25**

### CODE REVIEW

File: docs/en/observability/whats-new.asciidoc [hunk 1]
```diff
@@ -3,24 +3,94 @@
...
+image::images/2-nginx.png[slo entry point contextual awareness]
+
+[float]
+[[es-ql-editing-visualization]]
+=== ES|QL editing and visualization (tech preview)
+
+When the AI Assistant responses use ES|QL, you can now do the following:
+
+1. See the “Display results” of the ES|QL query
+2. Visualize the query on the generated ES|QL:
```
Comment (mdbirnstiehl) [inline docs/en/observability/whats-new.asciidoc:59]: Wasn't sure if this should be an unordered list or if it's actually a process of seeing the results then visualizing the query.

File: docs/en/observability/whats-new.asciidoc [hunk 2]
```diff
@@ -3,24 +3,94 @@
...
+* Elastic AI Assistant support for AWS Bedrock
+* Contextual awareness of the entry point into AI Assistant
+* Performance improvements
+* Tech preview of editing and visualizing ES|QL queries from the AI Assistant
+
+[float]
+[[aws-bedrock-support]]
+=== AWS Bedrock support
+
+Elastic Observability 8.13 adds connectivity for AWS Bedrock, specifically for the Claude models. The Bedrock/Claude LLM adapter parses data from AWS using a SerDe stream using `@smithy/eventstream-serde-nodeuses` and offers the same experience as the other Elastic LLM connectors (OpenAI and Azure OpenAI). Setup is simple with the Amazon Bedrock connector configuration page.
```
Comment (mdbirnstiehl) [inline docs/en/observability/whats-new.asciidoc:39]: Not sure about this, but I see `@smithy/eventstream-serde-node` online, but not `nodeuses`. However I don't really know anything about this particular instance, just looked off.

**Review by mdbirnstiehl — APPROVED** [review]
Just a couple of nits that you may or may not want to address, otherwise, LGTM

File: docs/en/observability/whats-new.asciidoc (hunk 1 above)
Comment (bmorelli25) [reply docs/en/observability/whats-new.asciidoc:59]:
```suggestion
* See the “Display results” of the ES|QL query
* Visualize the query on the generated ES|QL:
```

File: docs/en/observability/whats-new.asciidoc (hunk 2 above)
Comment (bmorelli25) [reply docs/en/observability/whats-new.asciidoc:39]:
```suggestion
Elastic Observability 8.13 adds connectivity for AWS Bedrock, specifically for the Claude models. The Bedrock/Claude LLM adapter parses data from AWS using a SerDe stream using `@smithy/eventstream-serde-node` and offers the same experience as the other Elastic LLM connectors (OpenAI and Azure OpenAI). Setup is simple with the Amazon Bedrock connector configuration page.
```

**Pull request merged**